game/world/triggers/movetrigger.cpp

```diff
@@ -155,7 +156,7 @@ void MoveTrigger::processTrigger(const TriggerEvent& e, bool onTrigger) {
       break;
       }
     case zenkit::MoverBehavior::OPEN_TIME: {
-      state = Open;
+      state = Close;
```

Comment:
This look very suspicious.
While such change may fix some particular cases, in overall it looks wrong to do `Close`, instead of open and vise-versa

Reply:
> I think this works because Open and Close do the opposite thing in tick function

That would mean, tick `tick` function is wrong and need to be changed, not state <--> MoverBehavior relation.

At this point probably would be fine to do auto-rename, such as
1. Close -> Else
2. Open -> Close
3. Else -> Open